includes/class-wp-job-manager-post-types.php

```diff
@@ -736,19 +738,40 @@ public static function get_permalink_structure() {
 			switch_to_locale( get_locale() );
 		}
 
-		$permalinks = wp_parse_args(
-			(array) get_option( 'wpjm_permalinks', array() ),
+		/**
+		 * Option `wpjm_permalink` was renamed to match other options in 1.32.0.
```

Comment:
`wpjm_permalink` is missing the 's'.

Reply:
Fixed in a5980321f66801dfc5d6e0805cd31f4edf15f2d1